modules/basics/src/main/java/com/opengamma/strata/basics/date/HolidayCalendar.java

```diff
@@ -280,7 +280,7 @@ public default int daysBetween(LocalDate startInclusive, LocalDate endExclusive)
    * Combines this holiday calendar with another.
    * <p>
    * The resulting calendar will declare a day as a business day if it is a
-   * business day in both source calendars.
+   * business day in either source calendar.
```

Comment:
I don't think this correction is correct. combinedWith is where a business day only happens if both calendars agree that it is a business day.

Reply:
The `CombinedHolidayCalendar.isHoliday()` method is `calendar1.isHoliday(date) || calendar2.isHoliday(date)` and the `LinkedHolidayCalendar.isHoliday()` method is `calendar1.isHoliday(date) && calendar2.isHoliday(date)` so I don't think this is correct? The changes here also line up with the existing JavaDoc for both CombinedHolidayCalendar and LinkedHolidayCalendar.

Reply:
The code refers to holidays, the Javadoc to business days.

Reply:
Ah okay, I did misread that, shall revert.